internal/testing/time.go

```diff
@@ -0,0 +1,64 @@
+package testing
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/go-openapi/strfmt"
+	"github.com/onsi/gomega/types"
+)
+
+// EqualTime creates a matcher that checks that the actual value is equal to the given expected value, even if they are
+// expressed using different time zones. Actual and expected values should be time.Time or strfmt.DateTime.
+func EqualTime(expected any) types.GomegaMatcher {
```

Comment:
Do we have enough uses of `strfmt.DateTime` to justify not using https://onsi.github.io/gomega/#betemporallycomparator-string-compareto-timetime-threshold-timeduration?

Alternatively does it make sense to wrap the existing matcher with a type conversion?

Reply:
I'd say we have many uses of `strfmt.DateTime` because that is what the OpenAPI code generator uses: all our "date-time" fields will be `strfmt.DateTime`. However in this particular pull request the new marcher is used only three times. I think it is convenient to have this kind of matchers to simplify tests, but it isn't a strong opinion. I will replace it with explicit type conversions and time zone changes.

Reply:
Done.